src/yb/util/wait_state.h

```diff
@@ -138,7 +138,7 @@ YB_DEFINE_ENUM_TYPE(
 
     // CQL Wait Events
     ((Parse, YB_CQL_WAIT_STATE))(Analyze)(Execute)(ExecuteWaitingForCB)
-    (CQLRead)(CQLWrite)
+    (CQLRead)(CQLWrite)(CQLHandling)(CQLHandlingDone)(FlushAsync)(FlushAsyncDone)
```

Comment:
We have FlushAsync in YBClient as well, which gets called from other places.

Can you prepend these to disambiguate: "ExecutorFlushAsync..."